From 6dfff96cecd7f8296492b4e8dbfc989121df6545 Mon Sep 17 00:00:00 2001
From: Steven Barrett <steven@liquorix.net>
Date: Tue, 7 Jun 2016 19:58:11 -0500
Subject: [PATCH] Revert "kbuild: Add option to turn incompatible pointer check
 into error"
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

This reverts commit ea8daa7b97842aab8507b5b5b1e3226cf2d514a6.

This commit caused the following error to occur during build.  Since
we're not necessarily developing a driver here at Zen Kernel sources,
lets just turn it back off.  In fact, on February 19th, Ingo reverted
this change just to have Daniel Wagner enable it again.

  CC [M]  drivers/char/lp.o
drivers/char/lp.c:823:12: error: initialization from incompatible pointer type [-Werror=incompatible-pointer-types]
  .write  = lp_console_write,
            ^
drivers/char/lp.c:823:12: note: (near initialization for ‘lpcons.write’)
---
 Makefile | 3 ---
 1 file changed, 3 deletions(-)

Index: linux-liquorix/Makefile
===================================================================
--- linux-liquorix.orig/Makefile
+++ linux-liquorix/Makefile
@@ -785,9 +785,6 @@ KBUILD_CFLAGS   += $(call cc-option,-Wer
 # Prohibit date/time macros, which would make the build non-deterministic
 KBUILD_CFLAGS   += $(call cc-option,-Werror=date-time)
 
-# enforce correct pointer usage
-KBUILD_CFLAGS   += $(call cc-option,-Werror=incompatible-pointer-types)
-
 # use the deterministic mode of AR if available
 KBUILD_ARFLAGS := $(call ar-option,D)
 
